src/pages/index.astro

```diff
@@ -3,192 +3,152 @@ import Main from '../layouts/Main.astro'
+			peers working toward social justice with data-driven research.
+		</p>
+	</section>
+	<section id="services">
+		<h2>
+			Our Services
+		</h2>
+		<div class="pillars">
+			<section>
+				<h3>
+					Economic Research
+				</h3>
+				<p>
+					We develop boutique research studies focusing on targeted issues
+					related to the intersection of policy, economics, and social justice.
+					This includes but is not limited to literature reviews, policy
+					recommendations, community consultation and cost/benefit analysis.
+					Our past work has included costing an alternative municipal budget
+					for the city of Ottawa with an emphasis on racial and gender equity,
+					research on Employment Insurance and best practices for post-secondary
+					education fees and debt cancellation.
+				</p>
+			</section>
+			<section>
+				<h3>
+					Training
+				</h3>
+				<p>
+					We provide personalized training on quantitative research methods
+					for teams, helping them build confidence and do their work more
+					efficiently. Vivic offers SPSD/M training for beginners and more
+					experienced users. Our introductory training "Getting Started:
+					Policy Analysis Using SPSD/M" gets researchers quickly up-and-running
+					with SPSD/M, saving time and frustration.
+				</p>
+			</section>
+			<section>
+				<h3>
+					Data Science
+				</h3>
+				<p>
+					We develop processes for procuring, organizing, and analyzing
+					information effectively and efficiently. We also provide our
+					clients with analysis they can easily replicate such as annual
+					factsheets or reports.
+				</p>
+			</section>
+		</div>
+		<OurWork/>
+	</section>
 </Main>
 
 <style>
-
-#maincontent {
-    font-family: var(--mainfont);
-    margin: 325px 0 0;
-}
-
 #services {
-    max-width: 1500px;
-    margin: 0 auto 75px auto;
+	max-width: 1500px;
+	margin: 0 auto 75px auto;
 }
 
 #hero {
-    margin: 80px 8% 155px 8%;
-    font-family: var(--mainfont);
-    width: 65%;
+	margin: 325px 8% 155px 8%;
+	width: 65%;
 }
 
 h1 {
-    font-family: var(--mainfont);
-    text-transform: uppercase;
-    font-weight: 200;
-    font-size: 50px;
-    letter-spacing: 4px;
-}
-#hero p {
-    width: 75%;
-    font-weight: 100;
-    font-size: 14pt;
+	font-weight: 200;
+	font-size: 3.2rem;
+	letter-spacing: 0.08em;
+	text-transform: uppercase;
 }
 
-.center-title {
-    text-align: center;
+#hero p {
+	width: 75%;
+	font-weight: 100;
+	font-size: 1.2rem;
 }
 
 h2 {
-    font-size: 30pt
-}
-
-.center-title {
-    text-align: center;
-}
-
-#pillars {
-    display: flex;
-    justify-content: space-evenly;
-    margin-bottom: 5rem;
-    z-index: -9;
-    -webkit-transform: translateZ(-1);
-    transform: translateZ(-1);
-}
-
-.pillar-box {
-    font-size: 18pt;
-    margin-top: 50px;
-    text-align: center;
-    padding: 10px 20px;
-    color: black;
-    font-weight: bold;
-    border-radius: 2px;
+	font-size: 2.3rem;
+	text-align: center;
```

Comment:
Including the center-alignment directly on the `<h2>` instead of the `.center-title` class, as it was the only user.